docs/monthly-meeting/2025-01.md

```diff
@@ -0,0 +1,100 @@
+
+
+## Discussion
+
+### Translations
+
+- Should we change the translation progress limit for being included in the language switcher?
+  - This is currently the tutorial and the bugs and functions pages.
+  - Brief discussion on this threshold, no strong desire to change.
+- [Adam] Introduce a page for translation progress, like [React's](https://translations.react.dev/), it just needs to be wired up.
+  - There's already [Maciej's page](https://m-aciek.github.io/pydocs-translation-dashboard/)
+  - Issue created: [docs-community#143](https://github.com/python/docs-community/issues/143)
+  - Action: link Maciej's page from the devguide.
+  - Action: Add links to contribute.
+  - Action: Look to move to `https://docs.python.org/translations` (via `docsbuild-scripts`)?
+- [Adam] Fate of the language switcher?
+  - Not very maintainable or good for accessibility. It might be better to have a page for choosing languages.
+  - Other docs are fairly evenly split between a page to choose and a language switcher per page.
+  - Switching to the *current* page in another language might be important.
+  - Seek feedback from those that use the switcher or frequently read non-English documentation.
+
+### Grammar documentation & Sphinx directives
+
+- Petr and Blaise have been working on an improved version of Sphinx's `production-list` directive to represent and document Python's grammar.
+- Petr provided a demonstration of his work.
+- There is an initial PR to add the directives at [#127835](https://github.com/python/cpython/pull/127835)
+- This initially adds the directive, grammar snippets (and the surrounding prose) will be converted later, potentially looking to make this automatic.
+
+### Docs for beginners
+
+- Can we break up the work so people can contribute in parallel?
+- Is this better on `docs.python.org`, or third parties (O'Reilly, w3schools, etc.)?
+  - Probably better *started* as a third-party effort, so you don't need approvals up-front. (That's also how additions to stdlib should work. Or the translation status page.)
+  - Third parties know their audience better, and can serve more diverse audiences.
+  - Tutorials need to cater to many different audiences (children in schools, scientists in labs, experienced programmers from other languages, those entirely new to programming), this might be better accomplished by third parties.
+- Many people start with the python.org tutorial, and some “bounce off” and go to another tutorial or leave Python entirely.
+  - The existing tutorial is targeted at experienced programmers.
+  - Link to other tutorials (without endorsing)?
+- Mariatta was driving the effort.
+
+
+### Python Project Documentation
+
+*Relating to `docs.python.org`, `peps.python.org`, `devguide.python.org`, etc.*
+
+- [bswck] Track upstream->downstream translation block update alerts
+  - Not discussed.
+
+
```

Comment:
We didn't discuss this, let's remove it from here, and leave it in the agenda for another time.
```suggestion
```